examples/compiler_selection/Older.sol

```diff
@@ -0,0 +1,12 @@
+//SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.7.0 <0.8.0;
```

Comment:
Please set this to exactly `0.7.6` to ensure that we're not just getting `0.7.x`.

Reply:
lame
```
Error: Source file requires different compiler version (current compiler is 0.7.6+commit.7338295f.Linux.g++) - note that nightly builds are considered to be strictly less than the released version
 --> examples/compiler_selection/Older.sol:2:1:
  |
2 | pragma solidity ==0.7.6;
```

But `pragma solidity >=0.7.6 <0.7.7;` works